tests/pass-dep/libc/libc-fs-flock.rs

```diff
@@ -0,0 +1,71 @@
+// Flock tests are separate since they don't in general work on a Windows host.
```

Comment:
The comment is confusing... CI seems to indicate that it works just fine on a windows host?

Reply:
Yeah, it was copied from another test and I forgot to update it after the emulation was implemented.